src/App.tsx

```diff
@@ -1,222 +1,137 @@
+    }
+  });
 
-          <div data-cy="Todo" className="todo">
-            <label className="todo__status-label">
-              <input
-                data-cy="TodoStatus"
-                type="checkbox"
-                className="todo__status"
-              />
-            </label>
+  const filteredList = useMemo(() => filterList(), [currentFilter, todos]);
 
-            <span data-cy="TodoTitle" className="todo__title">React</span>
-            <button
-              type="button"
-              className="todo__remove"
-              data-cy="TodoDeleteButton"
-            >
-              ×
-            </button>
-
-            <div data-cy="TodoLoader" className="modal overlay">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
+  return (
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
 
-          <div data-cy="Todo" className="todo">
-            <label className="todo__status-label">
-              <input
-                data-cy="TodoStatus"
-                type="checkbox"
-                className="todo__status"
-              />
-            </label>
+      <div className="todoapp__content">
+        <Header />
+        {todos.length !== 0 && (
+          <>
+            <TodoList
+              filteredList={filteredList}
+              completedTodo={completedTodo}
+              handleCompletedTodo={handleCompletedTodo}
+            />
 
-            <span data-cy="TodoTitle" className="todo__title">Redux</span>
+            <FooterTodo
+              todos={todos}
+              setCurrentFilter={setCurrentFilter}
+              islinkAll={islinkAll}
+              islinkActive={islinkActive}
+              islinkCompleted={islinkCompleted}
+              setValue={setValue}
+            />
+          </>
+        )}
+
+        {isError && (
+          <div
+            data-cy="ErrorNotification"
+            className={classnames(
+              'notification is-danger is-light has-text-weight-normal',
+              {
+                hidden: isHidden,
+              },
+            )}
+          >
             <button
+              aria-label="button"
+              data-cy="HideErrorButton"
               type="button"
-              className="todo__remove"
-              data-cy="TodoDeleteButton"
-            >
-              ×
-            </button>
-
-            <div data-cy="TodoLoader" className="modal overlay is-active">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
+              hidden
+              onClick={() => {
+                setisHidden(true);
+              }}
+              className="delete"
+            />
+            Unable to add a todo
+            <br />
+            Unable to delete a todo
+            <br />
+            Unable to update a todo
```

Comment:
Do you need to show all these errors at one time? Add a condition
And it shouldn't be as a part of the button title